zscript/SoundBindings/Toby_WeaponNameSoundBindings.txt

```diff
@@ -1,24 +1,18 @@
 //Toby
-//{"ActorClass":"Fist_TO", "SoundToPlay":"stats/doom/weapons/fist"}
-//{"ActorClass":"Chainsaw_TO", "SoundToPlay":"stats/doom/weapons/chainsaw"}
-//{"ActorClass":"Pistol_TO", "SoundToPlay":"stats/doom/weapons/pistol"}
-//{"ActorClass":"Shotgun_TO", "SoundToPlay":"stats/doom/weapons/shotgun"}
-//{"ActorClass":"SuperShotgun_TO", "SoundToPlay":"stats/doom/weapons/ssg"}
-//{"ActorClass":"Chaingun_TO", "SoundToPlay":"stats/doom/weapons/chaingun"}
-//{"ActorClass":"RocketLauncher_TO", "SoundToPlay":"stats/doom/weapons/rlauncher"}
-//{"ActorClass":"PlasmaRifle_TO", "SoundToPlay":"stats/doom/weapons/plasmagun"}
-//{"ActorClass":"BFG9000_TO", "SoundToPlay":"stats/doom/weapons/bfg"}
+{"ActorClass":"Fist_TO", "SoundToPlay":"stats/doom/weapons/fist"}
+{"ActorClass":"Chainsaw_TO", "SoundToPlay":"stats/doom/weapons/chainsaw"}
+{"ActorClass":"Pistol_TO", "SoundToPlay":"stats/doom/weapons/pistol"}
+{"ActorClass":"Shotgun_TO", "SoundToPlay":"stats/doom/weapons/shotgun"}
+{"ActorClass":"SuperShotgun_TO", "SoundToPlay":"stats/doom/weapons/ssg"}
+{"ActorClass":"Chaingun_TO", "SoundToPlay":"stats/doom/weapons/chaingun"}
+{"ActorClass":"RocketLauncher_TO", "SoundToPlay":"stats/doom/weapons/rlauncher"}
+{"ActorClass":"PlasmaRifle_TO", "SoundToPlay":"stats/doom/weapons/plasmagun"}
+{"ActorClass":"BFG9000_TO", "SoundToPlay":"stats/doom/weapons/bfg"}
 
 //Doom 2
 {"ActorClass":"Fist", "SoundToPlay":"stats/doom/weapons/fist"}
 {"ActorClass":"Chainsaw", "SoundToPlay":"stats/doom/weapons/chainsaw"}
 {"ActorClass":"Pistol", "SoundToPlay":"stats/doom/weapons/pistol"}
-{"ActorClass":"Shotgun", "SoundToPlay":"ann/shotgun"}
-{"ActorClass":"SuperShotgun", "SoundToPlay":"ann/ssg"}
-{"ActorClass":"Chaingun", "SoundToPlay":"ann/chaingun"}
-{"ActorClass":"RocketLauncher", "SoundToPlay":"ann/rlauncher"}
-{"ActorClass":"PlasmaRifle", "SoundToPlay":"ann/plasmagun"}
-{"ActorClass":"BFG9000", "SoundToPlay":"ann/bfg"}
```

Comment:
Are you sure this is correct?

Reply:
Thank you for letting me know. The sounds specified are calling from the Weapons Add-on, not the core. I will make the correction.